### DISCUSSION TITLE
How to properly save from HTML pages?

### DISCUSSION THREAD

**Discussion by silverwings15** [gildas-lormeau/SingleFileZ#97] [Q&A] [answered]
Hi, not sure if this is a bug so i've decided to create a discussion thread instead. I have some old .maff files (mozilla archive format) of defunct wordpress blogs that i want to convert to SingleFileZ zips. unfortunately there is no way to make such a direct conversion, so i've been trying the workaround of first converting the .maff files to 'uncompressed' .htmls (which work perfectly fine), then opening those htmls in my browser and running SFZ on them. however, this results in everything aside from the text content being discarded entirely. 

is there any way to make this conversion or am i just out of luck?

**Comment by gildas-lormeau** [comment, marked as answer]
Did you use a HTTP server to open your saved pages? This is required in order to save the resources because of these issues in Firefox:  
- https://bugzilla.mozilla.org/show_bug.cgi?id=1266960
- https://bugzilla.mozilla.org/show_bug.cgi?id=1246236

Depending on what you have already installed on your machine, you can easily run a HTTP server from the command line, see https://gist.github.com/willurd/5720255 for example.

**Reply by silverwings15** [reply]
works fantastically, thank you very much!

**Reply by gildas-lormeau** [reply]
You're welcome!
